test/fail_compilation/b16360.d

```diff
@@ -0,0 +1,39 @@
+/*
+REQUIRED_ARGS: -inline
+
+TEST_OUTPUT:
+---
+fail_compilation/b16360.d(12): Error: function `b16360.foo` cannot inline function
+fail_compilation/b16360.d(25): Error: function `b16360.bar` cannot inline function
+---
+*/
+
+pragma(inline, true)
```

Comment:
@ibuclaw is pragma inline supported by GDC too ?

Reply:
Yes, but this test won't fail when compiled by gdc.

Reply:
So I must add a version() for LDC and GDC is suppose ?

Reply:
I'm sure they more than capable of omitting this test file when mirroring it into their repository.